src/atlas/util/Rotation.cc

```diff
@@ -126,9 +126,9 @@ void Rotation::precompute() {
 
 Rotation::Rotation(const PointLonLat& south_pole, double rotation_angle) {
```

Comment:
Now that I've comprehensively gone through the changes, I have 3 remarks none of which is urgent or even to do for this PR really (because the changes would be bit-identical to the tests as they are):
1. there are 3 cases here where the antipode is calculated, it should be moved into a single function (maybe as part of PointLonLat)
2. there's currently no accounting here for lat < -90 or lat > +90, or lon >> 360 (much greater than) or lon << 0 (much less than), although that is now part of Atlas's correct behaviour
3. the tests changed their reference results, so I don't understand why the comparison tolerance also had to change (for me that'd be the only think requiring a revert to original here?) (@benjaminmenetrier)

The first two points can be adressed later on discussing possible eckit::geo changes. This PR is a positive change -- thanks! :-) (and we can go ahead for the sync release of course, sorry for the delay, but I'd like a comment on the point above re. the comparison tolerances)

Reply:
Thanks @pmaciel that is reassuring. @benjaminmenetrier why is the tolerance of 1.e-9 required as opposed to 1.e-10? Is this due to different architectures or optimisation levels?